src/Pages/Form/Components/SectionTitle.tsx

```diff
@@ -8,7 +8,7 @@ const useStyle = makeStyles((theme: Theme) =>
   createStyles({
     root: {
       width: "100%",
-      padding: "118px",
+      padding: "50px",
```

Comment:
I suggest to add marginBottom: 32px.

Reply:
Where to add.

Reply:
Replace this line loey kub because SectionTitle will be implemented on top of all sections.